CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.0.1
+
+* Updated code to adhere to
+[Effective Dart](https://dart.dev/guides/language/effective-dart).
```

Comment:
Be clear what you're changing. Created a bunch of new members and deprecated old ones, etc.